packages/suite/src/actions/suite/initAction.ts

```diff
@@ -51,7 +51,7 @@ export const init = () => async (dispatch: Dispatch, getState: GetState) => {
     }
 
     // 6. init backends
-    await dispatch(blockchainActions.init()).catch(err => console.error(err));
+    await dispatch(initBlockchainThunk()).catch(err => console.error(err));
```

Comment:
This `catch` won't work correctly when there are some errors thrown in `initBlockchainThunk`. The thunks generated by _createAsyncThunk_ will always return a resolved promise with either the **fulfilled** action object or **rejected** action object inside. So the error is simply not caught here.

This is the reason why there is `unwrap()` function used a few lines above for dispatching of `connectInitThunk()`.
 
```
try {
        // it is necessary to unwrap the result here because init calls async thunk from redux-toolkit which is always resolved
        // see more details here: https://redux-toolkit.js.org/api/createAsyncThunk#unwrapping-result-actions
        await dispatch(trezorConnectActions.connectInitThunk()).unwrap();
    } catch (err) {
        dispatch({ type: SUITE.ERROR, error: err.message });
        throw err;
    }
```

The first solution is to add a case to extra reducers like:

`.addCase(initBlockchainThunk.rejected` and then log this error as you are doing it right now, or use the same approach with `.unwrap()` as it is done for `trezorConnectActions.connectInitThunk()`.

Reply:
https://github.com/trezor/trezor-suite/pull/6140/commits/74d13ed8eca1360367887af42bf58f7ef5bc362e